cds-plugin.js

```diff
@@ -3,29 +3,69 @@
 
 const cds = require("@sap/cds");
 const cdsPackage = require("@sap/cds/package.json");
-const { initializeFeatures } = require("./src/singleton");
+const { initializeFeatures, getFeaturesKeys, getFeatureValue } = require("./src/singleton");
 
-const activate = async () => {
-  const envFeatureToggles = cds.env.featureToggles;
-  if (envFeatureToggles?.config || envFeatureToggles?.configFile) {
-    // TODO this is currently done in package.json, because "cds build" ignores it otherwise. However, it should happen
-    //  dynamically.
-    // cds.env.requires["FeatureService"] = { model: "@cap-js-community/feature-toggle-library" };
-
-    if (Array.isArray(envFeatureToggles.serviceAccessRoles)) {
-      cds.on("loaded", (csn) => {
-        if (csn.definitions.FeatureService) {
-          csn.definitions.FeatureService["@requires"] = envFeatureToggles.serviceAccessRoles;
-        }
-      });
+const FEATURE_KEY_REGEX = /\/fts\/([^\s/]+)$/;
```

Comment:
> I'd use `cds.env.features.folders` instead of hardcoding `fts`. Beware of the pseudo-globs it supports though, e.g. the default is `fts/*`.

From what I can see even in our internal Capire docs, the `cds.env.features.folders` behavior is undocumented. I will not implement based on something that is intransparent to the end-users. 

Some edge cases of this `folders` are also unclear to me. Just of the top if my head, I could imagine that you 
- might choose a subset of folders, i.e., `["fts/a", "fts/b"]` => this is sensible and would work fine

- might choose a different prefix name `"features/*"` => this would not currently work. it is a bit weird for me... why would you do that? and it shows the first thing that is unclear, is folders a glob-string, a comma separated string, an array of strings or any of those?

- might mix both `["fts/a", "features/a"]` => this creates an odd situation now, because the vector that is expected in `req.features` is _without_ the prefix, meaning it's impossible to turn on/off one feature without the same happening to the other.